src/scripts/GoogleTagManagerLoader.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @implements {GoogleTagManagerLoader}
+ * @memberof module:GoogleTagManager
+ */
+class GoogleTagManagerLoader {
+  constructor({ config }) {
+    this.gtmId = config.GOOGLE_TAG_MANAGER_ID;
+    this.gtmArgs = '';
+    if (config.GOOGLE_TAG_MANAGER_AUTH) {
```

Comment:
could you please add respective config in the env as well

Reply:
I'm not sure where else it needs to be added. It's an optional feature so the it should not be there in the default config.